perl-package/AI-MXNet/examples/get_sherlockholmes_data.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+echo
+echo "NOTE: To continue, you need to review the licensing of the data sets used by this script"
+echo "See https://catalog.ldc.upenn.edu/ldc99t42 for the licensing"
```

Comment:
Does this still match for the new dataset?

Reply:
Thanks for point out! I will change the license to Gutenberg one:
https://www.gutenberg.org/wiki/Gutenberg:The_Project_Gutenberg_License